### PR TITLE
Formatted all source files following the Go standards.

### PR DESCRIPTION
**Pull request by Ch3ck**

### CODE REVIEW

**Comment by skycoin** [comment]
https://github.com/skycoin/skycoin/pull/208/commits/5dca6c2572b044f2764b8c756b8b499a42397560
                        
revert this; this is supposed to be that way

https://github.com/skycoin/skycoin/pull/208/commits/abfcd6e9d586833a5002e24c2ff9522d7bdf76d7

This is a hex value. In a cryptographic constant table. It is not a type or misspelling. Revert this.

Then resubmit.

File: src/cipher/secp256k1-go/secp256k1-go2/z_consts.go [hunk 1]
```diff
@@ -929,7 +929,7 @@ var pre_g = []XY{
...
 	{X: Field{[10]uint32{0x02298958, 0x0096e3ee, 0x01d935ee, 0x031b821f, 0x008fc5d9, 0x00fc8f31, 0x0371d193, 0x01ae5fcb, 0x024de160, 0x000c7509}}, Y: Field{[10]uint32{0x018251cb, 0x0004641c, 0x032c44ae, 0x02d8b98a, 0x00c8e41d, 0x021556e6, 0x025180c9, 0x007498a1, 0x01c0a3aa, 0x00118182}}},
 	{X: Field{[10]uint32{0x0103ad2f, 0x0091eb97, 0x03b78a87, 0x007602a3, 0x03352d1d, 0x037060d2, 0x02b206d3, 0x011c9182, 0x030a8ff1, 0x0007d547}}, Y: Field{[10]uint32{0x0116a6e3, 0x03a8cfb0, 0x03b32ecc, 0x02d94255, 0x00fcb97a, 0x000afb8d, 0x0144f818, 0x02c5236e, 0x0006a4c8, 0x0009094f}}},
-	{X: Field{[10]uint32{0x01cf333e, 0x02daed70, 0x003a87aa, 0x01ccaabb, 0x00206a1f, 0x03fd6a31, 0x03343119, 0x02101cc4, 0x021d27f0, 0x00378b13}}, Y: Field{[10]uint32{0x0195170d, 0x0173f8a0, 0x01a71c5a, 0x00ab2cb5, 0x019226ca, 0x007e9263, 0x03f4a8e4, 0x0334e8c7, 0x03a140fe, 0x0037204e}}},
+	{X: Field{[10]uint32{0x01cf333e, 0x02dead70, 0x003a87aa, 0x01ccaabb, 0x00206a1f, 0x03fd6a31, 0x03343119, 0x02101cc4, 0x021d27f0, 0x00378b13}}, Y: Field{[10]uint32{0x0195170d, 0x0173f8a0, 0x01a71c5a, 0x00ab2cb5, 0x019226ca, 0x007e9263, 0x03f4a8e4, 0x0334e8c7, 0x03a140fe, 0x0037204e}}},
```
Comment (Ch3ck) [inline src/cipher/secp256k1-go/secp256k1-go2/z_consts.go]: @skycoin That I should revert this right?

File: src/consensus/blockstat.go [hunk 2]
```diff
@@ -292,7 +292,6 @@ func (self *BlockStat) try_add_hash_and_sig(
 
 	if !(action_update || action_skip || action_insert) {
 		panic("Inconsistent BlockStat::try_add_hash_and_sig()")
-		return -1
```
Comment (Ch3ck) [inline src/consensus/blockstat.go]: @skycoin This too?

File: src/consensus/blockstat.go [hunk 3]
```diff
@@ -573,7 +572,6 @@ func (self *BlockStatQueue) try_append_to_BlockStatQueue(
 
 	if !(action_update || action_skip || action_insert) {
 		panic("Inconsistent")
-		return -1
 	}
```
Comment (Ch3ck) [inline src/consensus/blockstat.go]: @skycoin this too?

File: src/mesh/transport/stub.go [hunk 4]
```diff
@@ -107,7 +107,6 @@ func (self *StubTransport) SendMessage(toPeer cipher.PubKey, message []byte, ret
 	if retChan != nil {
 		retChan <- nil
 	}
-	return nil
```
Comment (Ch3ck) [inline src/mesh/transport/stub.go]: Should I revert this too?

File: src/cipher/secp256k1-go/secp256k1-go2/z_consts.go [hunk 5]
```diff
@@ -8438,7 +8438,7 @@ var prec = [64][16]XY{
 	{
 		{X: Field{[10]uint32{0x02d86047, 0x02567d0e, 0x00bcaeff, 0x0010ea6e, 0x0379b53a, 0x0132a419, 0x01cca776, 0x020e9dc6, 0x010bd359, 0x00239ef3}}, Y: Field{[10]uint32{0x0060372a, 0x00011fa1, 0x028b3ea1, 0x00b91ff5, 0x0179e88e, 0x032a5451, 0x00310420, 0x0292ce50, 0x030b2a3d, 0x00042ddd}}},
 		{X: Field{[10]uint32{0x013f0351, 0x00a022fc, 0x009e777d, 0x0234f40e, 0x033bc15b, 0x025a5437, 0x010f3199, 0x01ca770d, 0x03aa195e, 0x000cecd6}}, Y: Field{[10]uint32{0x038d7418, 0x0140f332, 0x01b702bc, 0x03bef222, 0x00aa6560, 0x012354af, 0x00640362, 0x02fe1fe5, 0x0185640a, 0x00296280}}},
-		{X: Field{[10]uint32{0x001ead4b, 0x0221e681, 0x003c1ffe, 0x03eda46c, 0x023a75ed, 0x02bf9daf, 0x014734ef, 0x00cdfb1c, 0x000c8c8f, 0x002ff243}}, Y: Field{[10]uint32{0x02fedaed, 0x00b1bc21, 0x02e3e745, 0x03fccb82, 0x02fb468e, 0x01410b4d, 0x03a718dd, 0x037b4936, 0x01b1e09c, 0x001ea520}}},
+		{X: Field{[10]uint32{0x001ead4b, 0x0221e681, 0x003c1ffe, 0x03eda46c, 0x023a75ed, 0x02bf9daf, 0x014734ef, 0x00cdfb1c, 0x000c8c8f, 0x002ff243}}, Y: Field{[10]uint32{0x02fedead, 0x00b1bc21, 0x02e3e745, 0x03fccb82, 0x02fb468e, 0x01410b4d, 0x03a718dd, 0x037b4936, 0x01b1e09c, 0x001ea520}}},
```
Comment (Ch3ck) [inline src/cipher/secp256k1-go/secp256k1-go2/z_consts.go]: I guess with this too?

**Pull request closed without merging**